web-animations-1/Overview.bs

```diff
@@ -85,9 +85,9 @@ Former Editor: Alex Danilo 31960, Google, adanilo@google.com
 Former Editor: Tab Atkins 42199, Google, jackalmage@gmail.com
 Abstract: This specification defines a model for synchronization and timing
           of changes to the presentation of a Web page.
-          This specification also also defines an application programming interface
+          It also also defines an application programming interface (API)
```

Comment:
Can we s/also also/also/ at the same time?

Reply:
Yes, lol. I can't believe I missed that.